### ISSUE TITLE
BatchNorm2d affect performance in testing time?

### ISSUE THREAD

**Issue by BlueAnthony** [lzrobots/LearningToCompare_ZSL#6]
First, thank you for your awesome work!
Because I see the test_accuracy are different every execution when I repeat to execute  "python miniimagenet_test_one_shot.py -w 5 -s 1", I found that the feature of the same image from feature encoder network will be different.
Should BatchNorm2d be turned off or deactivated by something like model.eval() in testing phase? 
Or, I just miss something.
Thank you!

**Comment by BlueAnthony** [comment]
Sorry. I ask for the few-shot part, not zero-shot detection. My fault.
So, I close this issue.
